### PR TITLE
Blockquotes

### PR DESCRIPTION
**Pull request by gemfarmer**
Fixes issue(s) #2140 

[CircleCI](https://circleci.com/gh/18F/18f.gsa.gov/tree/blockquotes)

[:sunglasses: PREVIEW](https://federalist.18f.gov/preview/18F/18f.gsa.gov/blockquotes/)
[List of blockquotes](https://federalist.18f.gov/preview/18F/18f.gsa.gov/blockquotes/2016/11/25/what-18f-is-thankful-for-this-thanksgiving/)
[List of blockquote + image](https://federalist.18f.gov/preview/18F/18f.gsa.gov/blockquotes/2016/03/21/we-asked-100-of-our-coworkers-why-did-you-join-18f/)

Changes proposed in this pull request:
- adds styling for `<blockquote>`and `<quote>` element in a number of layout situations, including:
-- single quote + image to be inline and indented
-- single quote to have left border and indented
-- list of quotes to adopt single quote styling 
-- 18F logo for quotes without images in list quote blog posts 

/cc @elainekamlley

### CODE REVIEW

File: _plugins/redcarpet.rb [hunk 1]
```diff
@@ -28,6 +28,18 @@ def strip_dashes(str)
     end
   end
 
+  def block_quote(quote)
+    quotes = quote.split("\n\n")
+    if quotes.count > 1
+      blockquote = quotes.map do |q|
```
Comment (gemfarmer) [inline _plugins/redcarpet.rb]: I think it might be technically unnecessary, but definitely more clear

**Comment by gemfarmer** [comment]
This is open for review!

**Comment by awfrancisco** [comment]
The content doesn't seem to be properly displaying in [this preview](https://federalist.18f.gov/preview/18F/18f.gsa.gov/blockquotes/2016/11/25/what-18f-is-thankful-for-this-thanksgiving/). I know this post was listed in the original issue, but I don't think the original used block quotes. The quotes were just in normal body text. 

And I'm not sure if it's related to the blockquote styling, but a bunch of links seem to be broken in the [other post](https://federalist.18f.gov/preview/18F/18f.gsa.gov/blockquotes/2016/03/21/we-asked-100-of-our-coworkers-why-did-you-join-18f/), but the design looks much better. 

The new style does look good on the bottom of [this post](https://federalist.18f.gov/preview/18F/18f.gsa.gov/blockquotes/2016/08/31/kanban-for-government/)

**Comment by coreycaitlin** [comment]
It looks like some issues with the `redcarpet` plugin are causing some blockquotes to spit out `blockquote.to_s` instead, as Andre noted.

**Comment by gemfarmer** [comment]
@coreycaitlin @awfrancisco thanks for the reviews! I'm on it!

**Comment by gemfarmer** [comment]
@awfrancisco I think both the posts you mentioned have been fixed:

- [blockquote.to_s issue](https://federalist.18f.gov/preview/18F/18f.gsa.gov/blockquotes/2016/11/25/what-18f-is-thankful-for-this-thanksgiving/)
- [markdown formatting issue](https://federalist.18f.gov/preview/18F/18f.gsa.gov/blockquotes/2016/03/21/we-asked-100-of-our-coworkers-why-did-you-join-18f/)

**Comment by awfrancisco** [comment]
Looks like there's still one cloud.gov link rendering as `[cloud.gov](http://cloud.gov/)` in the second post, but otherwise they look great.

**Comment by gemfarmer** [comment]
@awfrancisco good catch! I looked through and fixed all other instances (on the three "we asked 100 of our coworkers..." posts) where we threw markdown into html blocks. Let me know if you find another instance like that!

**Comment by awfrancisco** [comment]
Thanks, and will do!

**Comment by coreycaitlin** [comment]
@gemfarmer redcarpet seems to be breaking up multiline/multi-graf blockquotes into separate blockquotes, which leads to this kind of thing:

screen shot 2016-12-30 at 11 02 03 am

Is it possible to treat that whole thing as one blockquote so it looks like this instead?

screen shot 2016-12-30 at 11 10 16 am

**Comment by gemfarmer** [comment]
@coreycaitlin hmm 🤔. So this looks like an unintended consequence of trying to solve a different problem. Previously, redcarpet was taking separate blockquotes and merging them. So something like this
```
> blockquote 1

> blockquote 2
```
would render as

> blockquote 1
blockquote 2

instead of 

> blockquote 1

> blockquote 2

The way that we forced redcarpet to do this probably wasn't thorough enough. Back to the drawing board. 

cc @elainekamlley

**Comment by gemfarmer** [comment]
@coreycaitlin ok, I think this last fix should wrap things up nicely. Instead of tinkering with the `redcarpet` renderer and extensions, I tried to upgrade to `kramdown` and I think the migration went _really_ smoothly. _Wayyy_ smoother than anticipated, really 💃 

Check it out here: 	9e330cd

kramdown is the default supported renderer for Jekyll 3.0.0 and above, so we are in good company there. Some benefits of kramdown:
- no blockquote weirdness 👏 
- we no longer need the extension `with_toc_data` to force headings to render with associated dash delimited id slugs. 👏 
- we no longer need the `smart` SmartyPants smart quotes extension 👏 
- it renders code blocks by default, and since we have a `rouge` stylesheet, there should be no migration issues here 👏 

I'm going to be clicking around to different pages thoughout the site, but so far, it looks like this is our solution!!! 🍾 🎆

**Pull request merged**